CONTRIBUTING.md

```diff
@@ -47,24 +51,22 @@ As of Go 1.9 this is simplified to `go test ./...`.
 
 ## Manual Testing against Exercism
 
-You can build whatever is in your local, working copy of the CLI without overwriting your existing Exercism
-CLI installation by using the `go build` command:
+To test your changes while doing everyday Exercism work you
+can build using the following instructions. Any name may be used for the
+binary (e.g. `testercism`) - by using a name other than `exercism` you
+can have different profiles under `~/.config` and avoid possibly
+damaging your real Exercism submissions, or test different tokens, etc.
 
-```
-go build -o testercism exercism/main.go
-```
-
-This assumes that you are standing at the root of the exercism/cli repository checked out locally, and it will put a binary named `testercism` in your current working directory.
-
-You can call it whatever you like, but `exercism` would conflict with the directory that is already there.
-
-Then you call it with `./testercism`.
+On Unices:
 
-You can always put this in your path if you want to run it from elsewhere on your system.
+- `cd $GOPATH/src/github.com/exercism/cli/exercism && go build -o testercism main.go`
+- `./testercism -h`
 
-We highly recommend spinning up a local copy of Exercism to test against so that you can mess with the database (and so you don't accidentally break stuff for yourself in production).
+On Windows:
 
-[TODO: link to the nextercism repo installation instructions, and explain how to reconfigure the CLI]
+- `cd /d %GOPATH%\src\github.com\exercism\cli`
+- `go build -o testercism.exe exercism\main.go`
+- `testercism.exe —help`
```

Comment:
Minor nitpick can we keep the flags consistent (`-h` or `—help`) they do the same thing.